src/orquestra/sdk/_base/_driver/_models.py

```diff
@@ -309,14 +309,26 @@ class CreateWorkflowRunResponse(pydantic.BaseModel):
 class ListWorkflowRunsRequest(pydantic.BaseModel):
     """
     Implements:
-    https://github.com/zapatacomputing/workflow-driver/blob/34eba4253b56266772795a8a59d6ec7edf88c65a/openapi/src/resources/workflow-runs.yaml#L9.
+    https://github.com/zapatacomputing/workflow-driver/blob/c52013c0f4df066159fc32ad38d489b3eaff5850/openapi/src/resources/workflow-runs.yaml#L14.
+
+    Returns all workflow runs along with their definition IDs, statuses and owners.
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏Technically this doesn't return anything.
If you want to reference the what is returned, you could add a "See also" but saying this "returns" is misleading.

Reply:
[e9ae973](https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/335/commits/e9ae97358d6fdc2c2597aaedbeab7e5321e83687)